docs/topics/Gateway.md

```diff
@@ -1258,8 +1259,9 @@ Returns an object based on the information in [Get Gateway](#DOCS_TOPICS_GATEWAY
 
 ###### Session Start Limit Structure
 
-| Field       | Type    | Description                                                        |
-| ----------- | ------- | ------------------------------------------------------------------ |
-| total       | integer | The total number of session starts the current user is allowed     |
-| remaining   | integer | The remaining number of session starts the current user is allowed |
-| reset_after | integer | The number of milliseconds after which the limit resets            |
+| Field           | Type    | Description                                                        |
+| --------------- | ------- | ------------------------------------------------------------------ |
+| total           | integer | The total number of session starts the current user is allowed     |
+| remaining       | integer | The remaining number of session starts the current user is allowed |
+| reset_after     | integer | The number of milliseconds after which the limit resets            |
+| max_concurrency | integer | The number of identify requests allowed per 5 seconds              |
```

Comment:
This doesn't capture the nuance though. Because really it's based on the shard `rate_limit = shard_id % max_concurrency`

So, if you had a max concurrency of 16, and 16 shards for example, you could start them all up at the same time.

Whereas if you had 32 shards, if you tried to start up shard 0 and 16 at the same time for example, it would not work. You can start shards 0-15 concurrently, then 16-31...